doxygen/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+find_package(Doxygen)
+if(${DOXYGEN_FOUND})
```

Comment:
You can just put
```
if(DOXYGEN_FOUND)
```
using `if(<variable>)` pattern.

Reply:
Done. Thanks for the reminder.